cvmfs/swissknife_sync.cc

```diff
@@ -595,7 +600,8 @@ int swissknife::CommandSync::Main(const swissknife::ArgumentList &args) {
   sync->Traverse();
 
   LogCvmfs(kLogCvmfs, kLogStdout, "Exporting repository manifest");
-  UniquePtr<manifest::Manifest> manifest(mediator.Commit());
+  const bool use_alt_path = params.voms_authz.size();
```

Comment:
please! 0 and 1 are somewhat boolean, but why not `! params.voms_authz.empty()`?